googledataprocauthenticator/test_google_auth.py

```diff
@@ -0,0 +1,411 @@
+# Copyright (c) 2015  aggftw@gmail.com
```

Comment:
Is this license intentional?

Reply:
Oh no this is not intentional it was what was at the top of sparkmagic's test file that I created a copy of then rewrote for google tests. I'll change this

Reply:
No this wasn't intentional. It was what was at the top of sparkmagic's test file that I created a copy of then rewrote for google tests. 

Actually about half of the tests are copied from a test file in Sparkmagic and for those copied tests I am using a Google Auth instance instead of just a string like "Kerberos" or "Basic" (what sparkmagic did before. So does this mean I should list that license as well? Not sure if I am allowed to delete it. Also for my upstream Sparkmagic PR I also didn't add any licenses at the top should I have?